EIPS/eip-2566.md

```diff
@@ -0,0 +1,128 @@
+---
+eip: 2566
+title: Human Readable Parameters for Contract Function Execution
+author: Joseph Stockermans (@jstoxrocky)
+discussions-to: https://ethereum-magicians.org/t/human-readable-parameters-for-contract-function-execution/4154, https://github.com/ethereum/EIPs/issues/2567
```

Comment:
Please pick a single location for discussion to occur.  We generally prefer Ethereum Magicians, so I recommend that one but either is valid.